mooncake-store/include/types.h

```diff
@@ -19,7 +19,8 @@ namespace mooncake {
 static constexpr uint64_t WRONG_VERSION = 0;
 static constexpr uint64_t DEFAULT_VALUE = UINT64_MAX;
 static constexpr uint64_t ERRNO_BASE = DEFAULT_VALUE - 1000;
-static constexpr uint64_t DEFAULT_DEFAULT_KV_LEASE_TTL = 200;  // in milliseconds
+static constexpr uint64_t DEFAULT_DEFAULT_KV_LEASE_TTL =
```

Comment:
TODO: make it configurable?

Reply:
https://github.com/kvcache-ai/Mooncake/blob/8ccf6932bd86250fd093968dcd6caae4fe048686/mooncake-store/src/master.cpp#L19C1-L20C50